lightning/log/log.go

```diff
@@ -106,6 +106,11 @@ func L() Logger {
 	return appLogger
 }
 
+// SetAppLogger replaces the default logger in this package to given one
+func SetAppLogger(l *zap.Logger) {
+	appLogger = Logger{l.WithOptions(zap.AddStacktrace(zap.DPanicLevel))}
```

Comment:
Why still need to change the log config here?

Reply:
keep same with https://github.com/pingcap/tidb-lightning/blob/963829fd54f16ba135790bc8bde4b8c2d48fb1aa/lightning/log/log.go#L96-L98

Reply:
They are not the same.  #96~98 is the default operation to build an app log. But if we build a logger and set it as global, the caller should take responsibility to set all options and lightning shouldn't override its config anymore.

Reply:
that's not override, just create a new logger for lightning's specification, and the comment remains reasonable IMO